server/Controllers/PointsController.cs

```diff
@@ -0,0 +1,103 @@
+using System;
+using System.Collections.Generic;
+using hutel.Filters;
+using hutel.Models;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.Extensions.Caching.Memory;
+
+namespace hutel.Controllers
+{
+    [Route("api/[controller]")]
+    public class PointsController : Controller
+    {
+        private IMemoryCache _memoryCache;
+        private const string _pointsKey = "points";
+        private const string _tagsKey = "tags";
+
+        public PointsController(IMemoryCache memoryCache)
+        {
+            _memoryCache = memoryCache;
+            Dictionary<Guid, Point> points;
+            if (!_memoryCache.TryGetValue(_pointsKey, out points))
+            {
+                _memoryCache.Set(
+                    _pointsKey,
+                    new Dictionary<Guid, Point>(),
+                    new MemoryCacheEntryOptions()
+                        .SetPriority(CacheItemPriority.NeverRemove)
+                );
+            }
+            Dictionary<string, Tag> tags;
+            if (!memoryCache.TryGetValue(_tagsKey, out tags))
+            {
+                _memoryCache.Set(
+                    _tagsKey,
+                    CreateTags(),
+                    new MemoryCacheEntryOptions()
+                        .SetPriority(CacheItemPriority.NeverRemove)
+                );
+            }
+        }
+
+        // POST api/points/id
+        [HttpPost("{id}")]
+        [ValidateModelState]
+        public IActionResult Post(Guid id, [FromBody]Point point)
+        {
+            var points = _memoryCache.Get<Dictionary<Guid, Point>>(_pointsKey);
```

Comment:
Please provide in comments or as simple test method with correct POST request example.
I was not able to figure out how to reach this post handler.
I tried diffferent conpinations e.g. POST http://localhost:5000/api/points/cd431eca-1b48-4dc3-b370-7961e0abcc3f with body {'Id': 123, 'TagId': 'testtag', 'Date':'12/21/2016', 'Extra': {} }.
I am getting either 400 BadRequest (where I reach code from ValidateModelStateAttribute) or 415 Unsupported media type.